spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/config/ConfigMapConfigProperties.java

```diff
@@ -82,10 +82,10 @@ public void setSources(List<Source> sources) {
 						"'spring.cloud.kubernetes.config.useNameAsPrefix' is set to 'true', but 'spring.cloud.kubernetes.config.sources'"
 								+ " is empty; as such will default 'useNameAsPrefix' to 'false'");
 			}
-			return Collections.singletonList(new NormalizedSource(name, namespace, ""));
+			return Collections.singletonList(new NormalizedSource(name, namespace, "", useProfileNameAsSuffix));
```

Comment:
we can't use `true` here, but need to pass it further. There is a test that explains this, but still: even if users do not specify any `sources` directly, there is one that gets created nevertheless. Because there are many profiles active, potentially, there could be many sources that would come from this single one. Users can still say "even for the default normalized source, I want to skip suffixing profile names". So we can't set `true` in here